packages/react-components/src/index.ts

```diff
@@ -55,7 +73,42 @@ export type {
   StrokeWidthTokens,
   Theme,
 } from '@fluentui/react-theme';
-export { SSRProvider } from '@fluentui/react-utilities';
+export {
+  clamp,
+  getNativeElementProps,
+  getNativeProps,
+  getPartitionedNativeProps,
+  getRTLSafeKey,
+  getSlots,
+  omit,
+  resolveShorthand,
+  shouldPreventDefaultOnKeyDown,
+  useBoolean,
+  useConst,
+  useControllableState,
+  useId,
+  useIsSSR,
+  useMergedEventCallbacks,
```

Comment:
```suggestion
```
I think this is very specific to library development since we need to merge multiple callbacks quite often in things like triggers. It might not be that useful for customers